fairscale/optim/grad_scaler.py

```diff
@@ -30,8 +30,22 @@ class ShardedGradScaler(TorchGradScaler):
     documentation https://pytorch.org/docs/stable/amp.html#torch.cuda.amp.GradScaler
     """
 
-    def __init__(self, process_group: Any = dist.group.WORLD) -> None:
-        super().__init__()
+    def __init__(
+        self,
+        init_scale: float = 2.0 ** 16,
+        growth_factor: float = 2.0,
+        backoff_factor: float = 0.5,
+        growth_interval: int = 2000,
+        enabled: bool = True,
+        process_group: Any = dist.group.WORLD,
+    ) -> None:
+        super().__init__(
+            init_scale=init_scale,
+            growth_factor=growth_factor,
+            backoff_factor=backoff_factor,
+            growth_interval=growth_interval,
+            enabled=enabled,
+        )  # type: ignore
```

Comment:
Looks great to me! Thanks for sending in the PR.

The type: ignore seems to be causing issue with py37 & mypy on CI. @ncilfone Can you see the CI error?

cc: @blefaudeux

Reply:
Yeah I see the error on my end when I look at the CircleCI details. mypy test runs cleanly locally for me (OSX + Python 3.8.5) with the `type: ignore` flag but that py37 CI build seemingly doesn't want to work...

Reply:
@min-xu-ai Looks like a single test of the ~500 is failing for gpu_tests_181-3 on CI as well... also seems a little out of the ordinary

Reply:
That's a timeout, might be a CI infra issue. I have triggered a rerun. The py37 issue might be real and not a flaky one. Once CI passes, I can merge this.

Reply:
Seems that the mypy error is real. Nicolas, I can't really checkout your branch and try to fix it. Maybe this is a py37 issue? Do you have 3.7 locally to try it? If not, I can dig it a bit and in the worse case, we can't disable mypy for 3.7.

Reply:
Working on debugging locally... will let you know

Reply:
@min-xu-ai New push should hopefully resolve the mypy issue. Didn't realize there were stubs for `torch.cuda.amp.GradScaler`. Adding a stub for `__init__` resolved the issue locally for me on Python 3.7.9

Reply:
@min-xu-ai Actually I've made a dumb mistake here by amending to my OG commit and force pushing to master on my fork. Seems like circle CI is used a cached version of the checkout (since it probably has the same SHA). Can you force a re-run without the cache? I can't on my end in CircleCI...